api-report/purchases-js.api.md

```diff
@@ -23,6 +23,9 @@ export interface CustomerInfo {
     readonly requestDate: Date;
 }
 
+// @public
+export type CustomTranslations = Record<string, Record<string, string>>;

```

Comment:
I think we don't need to export this anymore after removing the parameter?

Reply:
Yep, removing it.

Reply:
Done